config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```diff
@@ -731,3 +731,51 @@ periodics:
     testgrid-tab-name: sig-network-kind, detect-local-interface-name-prefix
     description: Runs network tests using KIND against latest kubernetes master with a kubernetes-in-docker cluster and kube-proxy detectLocalMode=InterfaceNamePrefix
     testgrid-alert-email: antonio.ojea.garcia@gmail.com, widaly@microsoft.com
+- interval: 12h
```

Comment:
ipvs is `6h`?

Reply:
things from the past, I should update all of them to daily to reserve resources, in other areas without presubmits getting more granularity is good, but during the last years there was no benefit of running them on short cycles, we don't use to merge code in this area as often